docs/spec/lcd/readme.md

```diff
@@ -0,0 +1,47 @@
+# Overview
+
+These documents describe the Light Client Daemon, commonly referred to as LCD. The LCD is split into
```

Comment:
This implies that the LCD has two separate components. Isn't it just one API? Maybe `the LCD API has two separate components`

Reply:
We need to rename LCD to sth. else as Tendermint-light-client and Cosmos-light-client can be confusing.

Reply:
Planned is to have the Cosmos light client rest server be the one primary interface, making the Tendermint light client transparent to the user/developer.

Reply:
what's the difference between cosmos-light-client and lcd?

Reply:
There are two LCDs is what I say. One for Tendermint and one for Cosmos.

Reply:
Are these two separate processes?

Reply:
There are two separate libraries which are going to be used to build the Cosmos Hub LCD. 

There is the Tendermint Light Client package. This package defines how to follow headers and update the root of trust. 

Then there will be the cosmos-sdk light client package. It imports the tendermint light client package and may re-export some of it's functionality. The sdk package defines how to verify proofs and uses some of the above package. 

Any developer that is building on top of the cosmos-sdk only uses the cosmos-sdk light client package. They never use the tendermint light client package directly.

We will use the cosmos-sdk light client package to build the LCD for the Cosmos hub. Zcash (an example, not real) will use the cosmos-sdk light client package to build the LCD for the zcash zone.